pkgs/by-name/ne/netmdplusplus/package.nix

```
@@ -0,0 +1,52 @@
+{
+  cmake,
+  fetchFromGitHub,
+  lib,
+  libgpg-error,
+  libgcrypt,
+  libusb1,
+  stdenv,
+  testers,
+}:
+stdenv.mkDerivation (finalAttrs: rec {
```

rec and finalAttrs provide the same functionality
```suggestion
stdenv.mkDerivation (finalAttrs: {
```

---

Thank you; removed unnecessary `rec`